src/ui/public/filter_bar/lib/map_geo_bounding_box.js

```diff
@@ -1,20 +1,33 @@
 import _ from 'lodash';
+import { SavedObjectNotFound } from '../../errors';
 
 export function FilterBarLibMapGeoBoundingBoxProvider(Promise, courier) {
   return function (filter) {
     if (filter.geo_bounding_box) {
-      return courier
-      .indexPatterns
-      .get(filter.meta.index).then(function (indexPattern) {
+      function getParams(indexPattern) {
         const type = 'geo_bounding_box';
         const key = _.keys(filter.geo_bounding_box)
           .filter(key => key !== 'ignore_unmapped')[0];
-        const field = indexPattern.fields.byName[key];
         const params = filter.geo_bounding_box[key];
-        const topLeft = field.format.convert(params.top_left);
-        const bottomRight = field.format.convert(params.bottom_right);
+        const topLeft = indexPattern
```

Comment:
I think a comment would be helpful:
`
indexPatterns are only used to find the field formatter for this field. An index pattern might not exist in some filters (... do you know how this can happen?), in which case we just display the raw value.
`

Up to you, but I found this code easy to read only after I read your comments on the PR, and a developer reading the code won't have that benefit. 

Or maybe a comment for this `getParams ` function at the top that says indexPattern is an optional parameter and used to format the field if given.